src/hooks/web3-constants.js

```diff
@@ -0,0 +1,142 @@
+import _CONTRACTS from 'https://contracts.webaverse.com/config/addresses.js';
+import _CONTRACT_ABIS from 'https://contracts.webaverse.com/config/abi.js';
```

Comment:
If we are importing the abi... then why are we also uploading the ABI to `app`?

We need to decide what we are doing and remove dead constants.

Reply:
It's a dead constant.
I removed it.